cmd/keytransparency-client/grpcc/grpc_client.go

```diff
@@ -206,12 +205,7 @@ func (c *Client) Update(ctx context.Context, userID, appID string, profileData [
 		return nil, fmt.Errorf("CreateUpdateEntryRequest: %v", err)
 	}
 
-	// Check the mutation before submitting it.
-	m, err := proto.Marshal(req.GetEntryUpdate().GetUpdate())
-	if err != nil {
-		return nil, fmt.Errorf("proto.Marshal(): %v", err)
-	}
-	if _, err := c.mutator.Mutate(getResp.GetLeafProof().GetLeaf().GetLeafValue(), m); err != nil {
+	if _, err := c.mutator.Mutate(getResp.GetLeafProof().GetLeaf().GetLeafValue(), req.GetEntryUpdate().GetUpdate()); err != nil {
```

Comment:
Changes to widely used APIS like `Mutate` typically touch a lot of files and are best processed as their own PRs.
Would you mind breaking this out?